app/src/main/java/org/schabi/newpipe/local/holder/LocalPlaylistStreamItemHolder.java

```diff
@@ -80,6 +81,10 @@ public void updateFromItem(final LocalItem localItem,
             itemDurationView.setVisibility(View.GONE);
         }
 
+        if (!DependentPreferenceHelper.getPositionsInListsEnabled(itemProgressView.getContext())) {
+            itemProgressView.setVisibility(View.GONE);
+        }
```

Comment:
I think you should add `itemProgressView.setVisibility(View.GONE)` below line 81 (don't know why it wasn't there before), and add the `getPositionsInListsEnabled` condition to the `if` on line 72.